src/Parser/DestructionWarlock/Modules/Talents/Eradication.js

```diff
@@ -0,0 +1,34 @@
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import Enemies from 'Parser/Core/Modules/Enemies';
+
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+
+import getDamageBonus from '../WarlockCore/getDamageBonus';
+
+const ERADICATION_DAMAGE_BONUS = 0.15;
+
+//only calculates the bonus damage, output depends if we have the talent directly or via legendary finger (then it appears as either a Statistic or Item)
```

Comment:
I recommend showing it as a statistic always since it's basically the talent that gives you the damage, and the ring is just an indirect way of getting it. This makes the display of the talent more consistent and predictable. In the item display the only real gain is the talent. Example of how this was solved elsewhere:

https://github.com/MartijnHols/WoWAnalyzer/blob/cfe1de5908bbaa288fe0fddc7f6a5a85001631fc/src/Parser/HolyPaladin/Modules/Items/SoulOfTheHighlord.js#L22

Reply:
Fair enough, I'll change it in next PR, along with the same issue in Affliction (Death's Embrace talent) :)